src/main/resources/config/application.properties

```diff
@@ -54,18 +54,18 @@ idam.client.redirect_uri=${IDAM_OAUTH2_REDIRECT_URL:https://localhost:3000/authe
 document_management.url=${DOCUMENT_MANAGEMENT_URL:http://dm-store:5005}
 dm_gateway.url=${DM_GATEWAY_URL:http://localhost:3455}
 doc_assembly.url=${DOC_ASSEMBLY_URL:http://localhost:4631}
-doc_assembly.adjourn_case=TB-SCS-GNO-ENG-00524.docx
+doc_assembly.adjourn_case=TB-SCS-GNO-ENG-adjournment-notice.docx
 doc_assembly.postponementrequest=TB-SCS-GNO-ENG-00754.docx
 docmosis.accessKey= ${PDF_SERVICE_ACCESS_KEY:}
 docmosis.convert.endpoint= ${PDF_SERVICE_CONVERT_URL:https://docmosis.aat.platform.hmcts.net/rs/convert}
 docmosis.health.endpoint= ${PDF_SERVICE_HEALTH_URL:https://docmosis.aat.platform.hmcts.net/rs/status}
 docmosis.uri=${DOCMOSIS_SERVICE_BASE_URL:https://docmosis.aat.platform.hmcts.net/rs/render}
 
-documents.english.DIRECTION_ISSUED=TB-SCS-GNO-ENG-00091.docx
-documents.english.DECISION_ISSUED=TB-SCS-GNO-ENG-00091.docx
+documents.english.DIRECTION_ISSUED=TB-SCS-GNO-ENG-draft-decision-notice.docx
```

Comment:
I can see that this draft decision notice is used in test classes so is this now meant to be the default in properties file or is this a mistake?